rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -923,6 +923,9 @@ bool VKGSRender::load_program()
 		}
 	}
 
+	properties.cs.attachmentCount = 1;
```

Comment:
Alright let me check this out as i remember debug layer complaint about the attachment count